tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -160,27 +172,19 @@ def test_requests_fail_and_no_description_returned(self, svc, respond_with):
         with pytest.raises(LTIOutcomesAPIError, match="LTI outcome request failed"):
             svc.read_result(self.GRADING_ID)
 
-    def test_it_gracefully_handles_RequestException(self, requests, svc):
-        requests.post.side_effect = RequestException
-
-        with pytest.raises(
-            LTIOutcomesAPIError, match="Error calling LTI Outcomes service"
-        ):
-            svc.read_result(self.GRADING_ID)
```

Comment:
This is now covered by `test_requests_fail_if_the_third_party_request_fails()` above. Previously `LTIOutcomesClient` had to worry about both error responses (4xx or 5xx) and `requests` exceptions like `ConnectionError` etc. `HTTPService` now deals with that and raises `HTTPError` in either case, so only one test is needed in `LTIOutcomesClient`.